sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md

```diff
@@ -5,11 +5,26 @@
 
 ## 3.5.0 (2021-05-24)
 ### New Features
+- Add `AADB2CTrustedIssuerRepository` to manage the trusted issuer in AAD B2C.
 - Upgrade to [spring-boot-dependencies:2.4.5](https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.4.5/spring-boot-dependencies-2.4.5.pom).
 - Upgrade to [spring-cloud-dependencies:2020.0.2](https://repo.maven.apache.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2020.0.2/spring-cloud-dependencies-2020.0.2.pom).
 - Enable property azure.activedirectory.redirect-uri-template.([#21116](https://github.com/Azure/azure-sdk-for-java/issues/21116))
+- Support creating `GrantedAuthority` by groupId and groupName for web application.([#20218])
+  ```yaml
+  user-group:
+      allowed-group-names: group1,group2
+      allowed-group-ids: <group1-id>,<group2-id>
+      enable-full-list: false  
+  ```
+  | Parameter           | Description                                             |
+  | ------------------- | ------------------------------------------------------------ |
+  | allowed-group-names | if `enable-full-list` is `false`, create `GrantedAuthority` with groupNames which belong to user and `allowed-group-names` |
+  | allowed-group-ids   | if `enable-full-list` is `false`, create `GrantedAuthority` with groupIds which belong to user and `allowed-group-ids` |
+  | enable-full-list    | default is `false`.<br> if the value is `true`, create `GrantedAuthority` only with user's  all groupIds, ignore group names|
```

Comment:
Why ignore group names? If the logic is ignore group names, `allow-all-group-ids-only` is a better property name.

Reply:
yes, I agree

Reply:
How about:
1. Delete `enable-full-list`? 
2. Use `allowed-group-ids = *` to implement function of `enable-full-list`.
3. Let `allowed-group-ids = *` can work together with `allowed-group-names`.

Reply:
As we discussed here: https://github.com/Azure/azure-sdk-for-java/issues/20218#issuecomment-827476074